AnkiDroid/src/main/java/com/ichi2/anki/workarounds/BottomSheetDialogFragmentFix.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ *  This program is free software; you can redistribute it and/or modify it under
+ *  the terms of the GNU General Public License as published by the Free Software
+ *  Foundation; either version 3 of the License, or (at your option) any later
+ *  version.
+ *
+ *  This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ *  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ *  PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with
+ *  this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.ichi2.anki.workarounds
+
+import android.content.Context
+import android.content.res.Configuration
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import com.ichi2.anki.convertDpToPixel
+
+// https://stackoverflow.com/questions/41591733
```

Comment:
The stackoverflow link is good and all, but some text describing the issue is better than
making folks click through and wade through a bunch of stackoverflow stuff

```suggestion
// Android material library BottomSheet has a bug where it is not expanded in landscape mode
// So any time we load the bottom sheet or the configuration changes, we check if we are in
// landscape mode, and if so, we force the sheet to be an expanded height
// https://stackoverflow.com/questions/41591733
```